lighthouse-core/test/audits/script-treemap-data-test.js

```diff
@@ -0,0 +1,301 @@
+      };
+      const rootNode = ScriptTreemapData.prepareTreemapNodes('some/prefix', sourcesData);
+      expect(rootNode).toMatchObject(
+         {
+           'children': [
+             {
+               'children': undefined,
+               'name': '/main.js',
+               'resourceBytes': 100,
+               'unusedBytes': 50,
+             },
+             {
+               'children': undefined,
+               'name': 'not/a.js',
+               'resourceBytes': 101,
+               'unusedBytes': 51,
+             },
+           ],
+           'name': 'some/prefix',
+           'resourceBytes': 201,
+           'unusedBytes': 101,
+         }
+      );
+
+      expect(rootNode.name).toBe('some/prefix');
+      expect(rootNode.resourceBytes).toBe(201);
+      expect(rootNode.unusedBytes).toBe(101);
+
+      const children = rootNode.children || [];
+      expect(children[0].name).toBe('/main.js');
+      expect(children[1].name).toBe('not/a.js');
+    });
+
+    it('nodes have unusedBytes data', () => {
+      const sourcesData = {
+        'lib/folder/a.js': {resourceBytes: 100, unusedBytes: 50},
+        'lib/folder/b.js': {resourceBytes: 101},
+        'lib/c.js': {resourceBytes: 100, unusedBytes: 25},
+      };
+      const rootNode = ScriptTreemapData.prepareTreemapNodes('', sourcesData);
+      expect(rootNode).toMatchObject(
+         {
+           'children': [
+             {
+               'children': [
+                 {
+                   'name': 'a.js',
+                   'resourceBytes': 100,
+                   'unusedBytes': 50,
+                 },
+                 {
+                   'name': 'b.js',
+                   'resourceBytes': 101,
+                 },
+               ],
+               'name': 'folder',
+               'resourceBytes': 201,
+               'unusedBytes': 50,
+             },
+             {
+               'name': 'c.js',
+               'resourceBytes': 100,
+               'unusedBytes': 25,
+             },
+           ],
+           'name': '/lib',
+           'resourceBytes': 301,
+           'unusedBytes': 75,
+         }
+      );
+    });
+
+    it('nodes have duplicates data', () => {
+      const sourcesData = {
+        /* eslint-disable max-len */
+        'lib/folder/a.js': {resourceBytes: 100, unusedBytes: 50},
+        'lib/node_modules/dep/a.js': {resourceBytes: 101, duplicatedNormalizedModuleName: 'dep/a.js'},
+        'node_modules/dep/a.js': {resourceBytes: 100, unusedBytes: 25, duplicatedNormalizedModuleName: 'dep/a.js'},
+        'lib/node_modules/dep/b.js': {resourceBytes: 101, duplicatedNormalizedModuleName: 'dep/b.js'},
+        'node_modules/dep/b.js': {resourceBytes: 100, unusedBytes: 25, duplicatedNormalizedModuleName: 'dep/b.js'},
+        /* eslint-enable max-len */
+      };
+      const rootNode = ScriptTreemapData.prepareTreemapNodes('', sourcesData);
+      expect(rootNode).toMatchObject(
+         {
+           'children': [
```

Comment:
I'm surprised eslint doesn't complain about these quoted properties...

Reply:
It only cares about consistency, I think